test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -4,7 +4,10 @@
 
```

Comment:
Same here, does this apply for hf images as well?

Reply:
yes, we don't support horovod with trcomp for pytorch